doc/install.md

```diff
@@ -139,4 +140,32 @@ make
 ```
 If the compilation finishes without error messages (except perhaps for some warnings), an executable program `ABACUS.mpi` will be created in directory `bin/`
 
+[back to top](#download-and-install)
+
+## Build and install ABACUS with CMake
+
+Check the cmake version on your machine
+```bash
+cmake --version
+```
+ABACUS requires the minimum cmake version `3.18`.
+
+You can specify the bin path of ABACUS binary to install by `CMAKE_INSTALL_PREFIX`.
+```bash
+cmake -B build -DCMAKE_INSTALL_PREFIX=${ABACUS_BIN_PATH}
+```
+You can provide root path of each dependent package if the package cannot be automatically found by cmake.
+Keys `LAPACK_DIR`, `SCALAPACK_DIR`, `ELPA_DIR`, `FFTW3_DIR`, `CEREAL_INCLUDEDIR`, `BOOST_INCLUDEDIR`, `MPI_CXX_COMPILER` and `MKL_DIR`. are currently available to specify.
+For example
+```bash
+cmake -B build -DFFTW3_ROOT=/opt/fftw3
+```
+
+If the cmake has executed successfully, then
+```bash
+cmake --build build
+cmake --install build
+```
+If no install prefix is specified, the binary will be installed to `/usr/local/bin/ABACUS` by default.
```

Comment:
abacus may be in lowercase
see https://github.com/deepmodeling/abacus-develop/blob/37c03daba8ad2c9bdf54a941c0a503115c6215c6/CMakeLists.txt#L37

Reply:
Lowercase is right, and is more common in other softwares' executable files.